packages/react-core/src/demos/examples/AlertGroup/AlertGroupDemo.tsx

```diff
@@ -0,0 +1,423 @@
+    );
+  };
+
+  const addInfoAlertNotification = () => {
+    handleNewNotification(
+      'Info alert notification',
+      'Info alert',
+      'info',
+      getUniqueId(),
+      getTimeCreated(),
+      'Info alert notification description'
+    );
+  };
+
+  const addWarningAlertNotification = () => {
+    handleNewNotification(
+      'Warning alert notification',
+      'Warning alert',
+      'warning',
+      getUniqueId(),
+      getTimeCreated(),
+      'Warning alert notification description'
+    );
+  };
+
+  const addDefaultAlertNotification = () => {
+    handleNewNotification(
+      'Default alert notification',
+      'Default alert',
+      'default',
+      getUniqueId(),
+      getTimeCreated(),
+      'Default alert notification description'
+    );
+  };
+
+  const onAlertGroupOverflowClick = () => {
+    removeAllAlerts();
+    setDrawerExpanded(true);
+  };
+
+  const onMaxDisplayedAlertsMinus = () => {
+    setMaxDisplayed(normalizeAlertsNumber(maxDisplayed - 1));
+  };
+
+  const onMaxDisplayedAlertsChange = (event: any) => {
+    setMaxDisplayed(normalizeAlertsNumber(Number(event.target.value)));
+  };
+
+  const onMaxDisplayedAlertsPlus = () => {
+    setMaxDisplayed(normalizeAlertsNumber(maxDisplayed + 1));
+  };
+
+  const normalizeAlertsNumber = (value: number) => Math.max(Math.min(value, maxAlerts), minAlerts);
+
+  const alertButtonClasses = ['pf-c-button', 'pf-m-secondary'].join(' ');
+
+  const alertButtonStyle = { marginRight: '8px' };
+
+  const notificationBadge = (
+    <NotificationBadge
+      variant={getUnreadNotificationsNumber() === 0 ? 'read' : 'unread'}
+      onClick={onNotificationBadgeClick}
+      aria-label="Notifications"
+    >
+      <BellIcon />
+    </NotificationBadge>
+  );
+
+  const notificationDrawerActions = [
+    <DropdownItem key="markAllRead" onClick={markAllNotificationsRead} component="button">
+      Mark all read
+    </DropdownItem>,
+    <DropdownItem key="clearAll" onClick={removeAllNotifications} component="button">
+      Clear all
+    </DropdownItem>,
+    <DropdownItem key="close" component="button" onClick={() => setDrawerExpanded(false)}>
+      Close
+    </DropdownItem>
+  ];
+
+  const notificationDrawerDropdownItems = (key: React.Key) => [
+    <DropdownItem key="markRead" component="button" onClick={() => markNotificationRead(key)}>
+      Mark as read
+    </DropdownItem>,
+    <DropdownItem key="action" component="button" onClick={() => removeNotification(key)}>
+      Clear
+    </DropdownItem>
+  ];
+
+  const notificationDrawer = (
+    <NotificationDrawer>
+      <NotificationDrawerHeader count={getUnreadNotificationsNumber()}>
+        <Dropdown
+          onSelect={onDropdownSelect}
+          toggle={
+            <KebabToggle
+              onToggle={isActive => onDropdownToggle('dropdown-toggle-id-0', isActive)}
+              id="dropdown-toggle-id-0"
+            />
+          }
+          isOpen={openDropdownKey === 'toggle-id-0'}
```

Comment:
```suggestion
          isOpen={openDropdownKey === 'dropdown-toggle-id-0'}
```

Currently the kebab dropdown for the notification drawer itself doesn't open.